css/projectsModal-style.css

```diff
@@ -0,0 +1,173 @@
+.popupContainer {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100%;
+  z-index: 2;
+}
+
+.projectPopup {
+  background: #fff;
+  width: 95%;
+  height: 95%;
+  padding: 1rem 2rem;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.popupHeader {
+  position: relative;
+}
+
+.popupTitle {
+  width: 90%;
+  color: #172b4d;
```

Comment:
- [ ] Hello @akezeth, Great effort working on your pop up modal, But currently i notice that the link of the `first card` and all the `pop-up` card image is broken and it not showing from my end. Kindly check how you have targeted its directory and be sure you are targeting the correct path.

- [ ] I will also urge you prioritize and stick with the Figma color and other necessary properties as this will give the confidence in your ability to replicate any given any design.
Also i would like you to check the `border-radius` of the `see-live` and `see-source` as it is not in the Figma design.

| Your Design 👇  | Figma Design 👇 |
| ------------- | ------------- |
| ![akl](https://user-images.githubusercontent.com/78634841/232547811-ee5cb024-10a7-4fd5-b4fe-383e03d9be03.png) | ![real](https://user-images.githubusercontent.com/78634841/232547890-cbcf0d1f-20a3-444f-bd4c-a2abecff573e.png) |